src/Helper/EmsFields.php

```diff
@@ -35,6 +35,7 @@ final class EmsFields
     const ASSET_CONFIG_BORDER_COLOR = '_border_color';
     const ASSET_CONFIG_WATERMARK_HASH = '_watermark_hash';
     const ASSET_CONFIG_GET_FILE_PATH = '_get_file_path';
+    const ASSET_EMSLINK_PREFIX = 'ems://asset:';
```

Comment:
Not sure with the name nor the location. What with EMSLINK_ASSET_PREFIX and EMSLINK_OBJECT_PREFIX in the EMS\CommonBundle\Common\EMSLink class?

Reply:
Done